pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -346,15 +401,28 @@ func (reconciler *ReconcileKieApp) createCustomObjects(object v1.CustomObject, c
 		_, _ = reconciler.createCustomObject(obj.Name, obj.Namespace, obj.DeepCopyObject(), &corev1.Secret{})
 	}
 	for _, obj := range object.RoleBindings {
-		obj.SetGroupVersionKind(rbac_v1.SchemeGroupVersion.WithKind("RoleBinding"))
+		obj.SetGroupVersionKind(rbacv1.SchemeGroupVersion.WithKind("RoleBinding"))
 		controllerutil.SetControllerReference(cr, &obj, reconciler.scheme)
 		obj.SetNamespace(cr.Namespace)
-		_, _ = reconciler.createCustomObject(obj.Name, obj.Namespace, obj.DeepCopyObject(), &rbac_v1.RoleBinding{})
+		_, _ = reconciler.createCustomObject(obj.Name, obj.Namespace, obj.DeepCopyObject(), &rbacv1.RoleBinding{})
 	}
 	for _, obj := range object.DeploymentConfigs {
 		obj.SetGroupVersionKind(oappsv1.SchemeGroupVersion.WithKind("DeploymentConfig"))
 		controllerutil.SetControllerReference(cr, &obj, reconciler.scheme)
 		obj.SetNamespace(cr.Namespace)
+		for ti, trigger := range obj.Spec.Triggers {
+			if trigger.Type == oappsv1.DeploymentTriggerOnImageChange {
+				if !reconciler.checkImageStreamTag(trigger.ImageChangeParams.From.Name, trigger.ImageChangeParams.From.Namespace) {
+					err := reconciler.createLocalImageTag(trigger.ImageChangeParams.From, cr)
+					if err != nil {
+						logrus.Error(err)
```

Comment:
What now? Should it bubble up an error and fail to deploy?

Reply:
i decided they DC should still deploy even if the imagestreamtag doesn't succeed. not sure what the benefit is of stopping the env deployment.

Reply:
Not sure... is it 100% that the DC will fail? If it is, I suppose the benefit is user experience, in that instead of going through DC deployment errors and being confused as to why it's not working, they would see the IS error and focus on fixing it.

Reply:
DC won't "fail" ... it'll create fine, just won't trigger deployment until imagestream exists. On the plus side, openshift UI will give clues as to why the deploy hasn't started.